scripts/install_third_party.py

```diff
@@ -341,6 +341,42 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
+def install_elasticsearch_dev_server():
+    """This installs a local ElasticSearch server to the oppia_tools
+    directory to be used by development servers and backend tests.
+    """
+
+    try:
+        subprocess.call(
+            [common.ES_PATH, '--version'],
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE)
+        python_utils.PRINT('ElasticSearch is already installed.')
+    except OSError:
+        python_utils.PRINT('Installing ElasticSearch...')
+
+        if common.is_mac_os() or common.is_linux_os():
```

Comment:
[optional] Use `return` after line 355 and de-indent line 359-377 by 4. (To reduce nested blocks, reduce expect block size etc.)

Reply:
Done